src/quo2/components/list_items/preview_list.cljs

```diff
@@ -0,0 +1,100 @@
+                                 :rounded  8}
+                 :hole-radius   {:circular 13
+                                 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     26
+                 :hole-x        19
+                 :hole-y        -1}
+             16 {:border-radius {:circular 8
+                                 :rounded  8}
+                 :hole-radius   {:circular 9
+                                 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     18
+                 :hole-x        11
+                 :hole-y        -1}})
+
+(defn list-item [idx item list-size margin-left hole-size hole-radius hole-x hole-y]
+  (let [last-item? (= idx (- list-size 1))]
+    [rn/hole-view {:style {:margin-left   (if (= idx 0) 0 margin-left)}
+                   :holes (if last-item? []
+                              [{:x             hole-x
+                                :y             hole-y
+                                :width         hole-size
+                                :height        hole-size
+                                :borderRadius  hole-radius}])}
+     item]))
+
+(defn get-overflow-color [transparent? transparent-color light-color dark-color]
+  (if transparent?
+    transparent-color
+    (colors/theme-colors light-color dark-color)))
+
+(defn overflow-label [label size transparent? border-radius margin-left]
+  [rn/view {:style {:width            size
+                    :height           size
+                    :margin-left      margin-left
+                    :border-radius    border-radius
+                    :justify-content  :center
+                    :align-items      :center
+                    :background-color (get-overflow-color
+                                       transparent?
+                                       colors/white-opa-10
+                                       colors/neutral-20
+                                       colors/neutral-70)}}
+   (if (= size 16)
+     [quo2.icons/icon :main-icons2/more {:size 12
+                                         :color (get-overflow-color
+                                                 transparent?
+                                                 colors/white-opa-70
+                                                 colors/neutral-50
+                                                 colors/neutral-40)}]
+     [quo2.text/text {:size   (if (= size 32) :paragraph-2 :label)
+                      :weight :medium
+                      :style  {:color       (get-overflow-color
+                                             transparent?
+                                             colors/white-opa-70
+                                             colors/neutral-60
+                                             colors/neutral-40)
+                               :margin-left -2}}
+      (if (< label 100) (str "+" label) (i18n/label :counter-99-plus))])])
```

Comment:
What is 100? If it is a maximum value it would be nice to add a comment to improve code readability

Reply:
Or use a constant

Reply:
Added comment, thank you